.travis.yml

```diff
@@ -0,0 +1,28 @@
+
+language: node_js
+node_js:
+  - "8"
+cache:
+  directories:
+  - node_modules
+
+install:
+  - npm install -g dredd
+  - npm install
+
+script:
+  - npm test
+
+after_success:
+  - docker run --volume $(pwd)/src/docs:/home/node/apiary:Z giovannicuriel/aglio -i /home/node/apiary/apiary.apib -o - > ./apiary_${TRAVIS_BRANCH}.html
+  - chmod +x travis/publish.sh
+  - docker build -t ${TRAVIS_REPO_SLUG} -f DockerFile .
+  - travis/publish.sh
+
+deploy:
+  provider: pages
```

Comment:
The last time I tried Github Pages deployment, I could not find a way to just add the new rendered .apib and left all the other versions intact (so we would have all the rendered published versions of it). Instead, it would remove everything and only add the newest version. Did you notice something similar?